packages/web-components/fast-components/src/radio/radio.md

```diff
@@ -0,0 +1,109 @@
+- [Ant Design](https://ant.design/components/radio/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/radio)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/radio-button)
+---
+
+### API
+Extends [form associated custom element](../form-associated/form-associated-custom-element.md).
+
+*Component Name*
+`fast-radio`
+
+*IDL attributes*
+- `checked: boolean`
+  - The current checked state of the radio
+
+*Content attributes*
+- `readonly`
+  - The radio should be submitted with the form but should not be editable.
+- `disabled`
+  - The radio should be disabled from user interaction and will not be submitted with the form data.
+- `value` - Not visible to the user, it's used for form data and to distinguish between other radio buttons of the same name attribute value.
+- `name` - All radio buttons with the same name will operate as a group allowing only 
+one radio within the group to be selected at a time.
+- `checked`
+  - The initial checked value. 
+  
+*Events*
+- `change: CustomEvent`
+  - no custom data
+  - bubbles
+
+### Anatomy and Appearance
+
+```HTML
+<!-- shadow root -->
+<template 
+  role="radio"
+>
+  <label part="label"><slot></slot></label>
+  <div part="radio">
+    <div part="checked-indicator"></div>
+  </div>
+</template>
+<!-- end shadow root -->
+```
+
+*Slot Names*
+- default: label for the radio
+
+*Host Classes*
+- checked
+- disabled
+- required
+- readonly
+
+*Slotted Content/Slotted Classes*
+*CSS Parts*
+- radio
+- label
+- checked-indicator
+
+### States
+**checked**: `true` or `false`
+The checked state can be toggled by:
+- Clicking the radio button (or any of it's labels)
+- Pressing the space-bar while focus is placed on the radio button
+- Adding / removing the "checked" content attribute
+  - This will only trigger a change if the "checked" property has not been changed, either through user action or programmatically
+
+**disabled**: `true` or `false`
+When disabled, the value will not be changeable through user interaction. It should also not expose it's value to a form submission.
+
+**readonly**: `true` or `false`
+When readonly, the value will not be changeable through user interaction. The value will still be exposed to forms on submission.
+
+### Accessibility
+The root element inside the shadow-dom of the radio will be a focusable element with the following accessibility content attributes:
+[MDN Web docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_radio_role)
+- role: radio
+- aria-checked: the checked state of the component
+- aria-required: the required state of the component
+- aria-disabled: the disabled state of the component
+- tabindex: 0
+
+Because we're delegating focus to this element, the relevant information will be presented to AT users when they focus this element.
```

Comment:
are we delegating focus? I think we're actually focusing this element rather than delegating focus to an element within the markup.

Reply:
removed